DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/storm_control/test_storm_control_broadcast_traffic.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import asyncio
+
+from dent_os_testbed.utils.test_utils.cleanup_utils import cleanup_kbyte_per_sec_rate_value
+from dent_os_testbed.lib.devlink.devlink_port import DevlinkPort
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from random import randrange
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_get_loss
+)
+
+pytestmark = [
+    pytest.mark.suite_functional_storm_control,
+    pytest.mark.asyncio,
+    pytest.mark.usefixtures('cleanup_bridges', 'cleanup_tgen')
+]
+
+
+# set kbyte_per_sec_rate value
+async def devlink_set_value(dev_name, name, value, cmode, device_host_name):
+    out = await DevlinkPort.set(input_data=[{device_host_name: [
+        {'dev': f'{dev_name}', 'name': f'{name}', 'value': f'{value}', 'cmode': f'{cmode}'}]}])
```

Comment:
Why do we need all these f-strings? I believe it can be simply 'dev': dev_name, etc.